core/src/main/java/bio/terra/pearl/core/service/export/EnrolleeImportService.java

```diff
@@ -146,6 +151,8 @@ public Import importEnrollees(String portalShortcode, String studyShortcode, Stu
         if (ImportFileFormat.CSV.equals(fileFormat)) {
             exportOptions = IMPORT_OPTIONS_CSV;
         }
+        StudyEnvironmentConfig studyEnvConfig = studyEnvironmentConfigService.findByStudyEnvironmentId(studyEnv.getId());
+        exportOptions.setTimeZone(studyEnvConfig.getTimeZone());
```

Comment:
I'm not entirely sure if I feel this is correct. I like that you can change timezone for export, but it would be cool if import prompted you to specify what timezone incoming dates are e.g. with A-T using UTC, and perhaps only take the default if timeZone isn't specified.
```suggestion
        if (exportOptions.getTimeZone() == null) {
                StudyEnvironmentConfig studyEnvConfig = studyEnvironmentConfigService.findByStudyEnvironmentId(studyEnv.getId());
                exportOptions.setTimeZone(studyEnvConfig.getTimeZone());
        }
```

Reply:
Yup that idea definitely makes sense, it would be nice to be able to say "this import is in a specific timezone".  Was just beyond the scope of this feature.  

What we'll want to do for A-T is set their study timezone to UTC before we import anything, and then everything will work, and their exports will be consistent with their past exports.